### PR TITLE
chore: fix qlik eslint

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: src/hooks/use-render.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react-hooks/rules-of-hooks */
```
Comment (cbt1) [inline src/hooks/use-render.ts:1]: Do you know why these could now be removed?

File: src/pivot-table/components/cells/EmptyCell.tsx [hunk 2]
```diff
@@ -25,4 +25,8 @@ const EmptyCell = ({ style, index, isLastRow, isLastColumn, showLastRowBorderBot
   );
 };
 
+EmptyCell.defaultProps = {
```
Comment (cbt1) [inline src/pivot-table/components/cells/EmptyCell.tsx:28]: I think we a better off changing the index to be required.

File: src/hooks/use-render.ts (hunk 1 above)
Comment (enell) [reply src/hooks/use-render.ts:1]: That disable isn't needed on main either. I think that is a leftover from some other refactoring

**Review by cbt1 — APPROVED** [review]
LGTM, we should update the prettier config and tsconfig after this

**Comment by niekvanstaveren** [comment]
> LGTM, we should update the prettier config and tsconfig after this

Yes, that was my plan as well. Did not want to get this PR too big

**Pull request merged**